README.md

```diff
@@ -28,19 +26,19 @@ including configurations for various project settings.
 
 ## Usage
 
-1. **Install Cruft** (if you haven't already):
+1.**Install Cruft** (if you haven't already):
 
 ```sh
 pip install cruft
 ```
 
-2. **Generate a New Project**:
+2.**Generate a New Project**:
```

Comment:
Either use title case in all bulletpoints or in none of them (preferred). So better:

> Generate a new project